jplag.endToEndTesting/src/test/java/jplag/endToEndTesting/javaTestCases.java

```diff
@@ -0,0 +1,181 @@
+package jplag.endToEndTesting;
```

Comment:
Fix the package declaration of all classes. 

Should be something like `de.jplag.end_to_end_testing`

Reply:
change the package name from "endToEndTesting" to "end_to_end_testing". also moved the javaTestCases.java to the above mentioned package.

should I then also adapt the module name to "end-to-end-testing" ?

Reply:
I would leave the module name as it is. Therefore, I resolve the conversation here :)